modules/scalar_transport/src/bcs/UnaryRecombinationBC.C

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "UnaryRecombinationBC.h"
+
+registerMooseObject("ScalarTransportApp", UnaryRecombinationBC);
+
+InputParameters
+UnaryRecombinationBC::validParams()
+{
+  auto params = ADIntegratedBC::validParams();
```

Comment:
do we really need that? it s the same as binary with the same parameter

Reply:
@loganharbour what do you mean by the 👎 ?

Reply:
I mean that I see the difference and am fine with the two objects